JavaFiddle-ejb/src/main/java/ru/javafiddle/execution/test/CompileAndRunTest.java

```diff
@@ -0,0 +1,52 @@
+package ru.javafiddle.execution.test;
+
+import org.junit.Test;
+import ru.javafiddle.core.ejb.CompileAndRunBean;
+
+import static org.junit.Assert.fail;
+
+/**
+ * Created by nk on 08.04.2016.
+ */
+public class CompileAndRunTest {
+
+    @Test
+    public void CompileAndRunTest() {
+        CompileAndRunBean compileAndRunBean = null;
+        try {
+            compileAndRunBean = new CompileAndRunBean();
+        } catch (NullPointerException e) {
+            e.printStackTrace();
+        }
+        try {
+            System.out.println(compileAndRunBean.compile("Hello"));
```

Comment:
Looks ugly.

1) There should be no try/catch block of such kind.
2) The test should verify the results (use JUnit Assert API).